packages/core-ethereum/src/types/balance.ts

```diff
@@ -9,6 +10,10 @@ class Balance extends UINT256 implements Types.Balance {
   static get DECIMALS(): number {
     return 18
   }
+
+  public toFormattedString(): string {
+    return moveDecimalPoint(this.toString(), Balance.DECIMALS * -1) + ' ' + Balance.SYMBOL
```

Comment:
this one looks pretty insecure. What about using `` `${} ... ` ``?

Reply:
insecure?